ios/RNSScreenStack.mm

```diff
@@ -156,66 +206,40 @@ - (void)presentationControllerDidDismiss:(UIPresentationController *)presentatio
         }
       });
     }
+#endif
   }
 }
 
-- (void)markChildUpdated
-{
-  // do nothing
-}
-
-- (void)didUpdateChildren
-{
-  // do nothing
-}
-
-- (void)insertReactSubview:(RNSScreenView *)subview atIndex:(NSInteger)atIndex
-{
-  if (![subview isKindOfClass:[RNSScreenView class]]) {
-    RCTLogError(@"ScreenStack only accepts children of type Screen");
-    return;
-  }
-  subview.reactSuperview = self;
-  [_reactSubviews insertObject:subview atIndex:atIndex];
-}
-
-- (void)removeReactSubview:(RNSScreenView *)subview
-{
-  subview.reactSuperview = nil;
-  [_reactSubviews removeObject:subview];
-}
-
 - (NSArray<UIView *> *)reactSubviews
 {
   return _reactSubviews;
 }
 
-- (void)didUpdateReactSubviews
-{
-  // we need to wait until children have their layout set. At this point they don't have the layout
-  // set yet, however the layout call is already enqueued on ui thread. Enqueuing update call on the
-  // ui queue will guarantee that the update will run after layout.
-  dispatch_async(dispatch_get_main_queue(), ^{
-    self->_hasLayout = YES;
-    [self maybeAddToParentAndUpdateContainer];
-  });
-}
-
 - (void)didMoveToWindow
 {
+#ifdef RN_FABRIC_ENABLED
+  [super didMoveToWindow];
```

Comment:
`[super didMoveToWindow];` can be outside `ifdef`

Reply:
+1